R/calcSteel_Projections.R

```diff
@@ -1194,6 +1215,36 @@ calcSteel_Projections <- function(subtype = 'production',
         arrange('scenario', 'region', 'iso3c', 'year', 'variable')
     }
 
+    # merge inida expert guess
+    if (do_use_expert_guess) {
+      IEA_ETP_matched <- bind_rows(
+        IEA_ETP_matched %>%
+          filter('IND' != .data$iso3c | 'SSP2IndiaHigh' != .data$scenario),
+
+        IEA_ETP_matched %>%
+          filter('IND' == .data$iso3c & 'SSP2IndiaHigh' == .data$scenario) %>%
```

Comment:
Should be `&&`.  See above.

Reply:
But teacher, `||` throws
```
Error in filter(., "IND" != .data$iso3c || "SSP2IndiaHigh" != .data$scenario) : 
  ℹ In argument: `"IND" != .data$iso3c || "SSP2IndiaHigh" !=
  .data$scenario`.
Caused by error in `"IND" != .data$iso3c || "SSP2IndiaHigh" != .data$scenario`:
! 'length = 662340' in coercion to 'logical(1)'
```

Lucky for me that you also [teach](https://github.com/pik-piam/mrindustry/blob/149b45b1d08bc9eabe496fd8122dc458de397f1a/R/calcSteel_Projections.R#L399) [by](https://github.com/pik-piam/mrindustry/blob/149b45b1d08bc9eabe496fd8122dc458de397f1a/R/calcSteel_Projections.R#L452) [example](https://github.com/pik-piam/mrindustry/blob/149b45b1d08bc9eabe496fd8122dc458de397f1a/R/calcSteel_Projections.R#L618).

Reply:
I rend my garments in shame.